.github/workflows/ci.yml

```
@@ -0,0 +1,174 @@
+
+      - name: Set deployment environment variables
+        id: set_vars
+        run: |
+          if [ "${{ github.ref_name }}" = "main" ]; then
+            echo "BASE_URL=https://sma-heat-policy.sydney.edu.au/" >> $GITHUB_ENV
+          else
+            echo "BASE_URL=https://extreme-heat-tool-test-987661761927.asia-southeast1.run.app/" >> $GITHUB_ENV
+          fi
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)